src/components/Container/Container.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../styles/_partials/index';
+
+.Container {
```

Comment:
Is it worth keeping this around?

Reply:
No. I'll delete it. If there's ever a time in the future when we want it, I'll add it then.